processor/transformer/transformer.go

```diff
@@ -591,7 +603,7 @@ func (trans *handle) destTransformURL(destType string) string {
 	destinationEndPoint := fmt.Sprintf("%s/v0/destinations/%s", trans.config.destTransformationURL, strings.ToLower(destType))
 
 	if _, ok := warehouseutils.WarehouseDestinationMap[destType]; ok {
-		whSchemaVersionQueryParam := fmt.Sprintf("whSchemaVersion=%s&whIDResolve=%v", trans.conf.GetString("Warehouse.schemaVersion", "v1"), warehouseutils.IDResolutionEnabled())
+		whSchemaVersionQueryParam := fmt.Sprintf("whIDResolve=%t", trans.conf.GetBool("Warehouse.enableIDResolution", false))
```

Comment:
How does `whSchemaVersion` relate to transformation ?

Reply:
Based on this, we used to decide whether we want to have v0/util or v1/util. Currently, we are only supporting v1 now. 
https://github.com/rudderlabs/rudder-transformer/pull/3752/files#diff-bacfe9b0c30628b991e1ec726e1484fa5380fdedab72a9fdda99799906e8de60